### PR TITLE
refac: renaming of added grid loss and system correction

### PR DESCRIPTION
**Pull request by johevemi**
## Description

Renames added_system_correction to negative_grid_loss and added_grid_loss to positive_grid_loss

## References

* #907

### CODE REVIEW

File: source/databricks/calculation-engine/tests/integration/calculator/steps/aggregation/test_added_grid_loss.py
Comment (JesperGomes) [inline source/databricks/calculation-engine/tests/integration/calculator/steps/aggregation/test_added_grid_loss.py:1]: rename file?

File: source/databricks/calculation-engine/tests/integration/calculator/steps/aggregation/test_added_system_correction.py
Comment (JesperGomes) [inline source/databricks/calculation-engine/tests/integration/calculator/steps/aggregation/test_added_system_correction.py:1]: rename file?

File: source/databricks/calculation-engine/tests/integration/calculator/steps/aggregation/test_added_grid_loss.py
Comment (johevemi) [reply source/databricks/calculation-engine/tests/integration/calculator/steps/aggregation/test_added_grid_loss.py:1]: ah yes will do

Comment (johevemi) [reply source/databricks/calculation-engine/tests/integration/calculator/steps/aggregation/test_added_grid_loss.py:1]: done

File: source/databricks/calculation-engine/tests/integration/calculator/steps/aggregation/test_added_system_correction.py
Comment (johevemi) [reply source/databricks/calculation-engine/tests/integration/calculator/steps/aggregation/test_added_system_correction.py:1]: done

File: source/databricks/calculation-engine/package/schemas/grid_loss_sys_corr_schema.py
Comment (JesperGomes) [inline source/databricks/calculation-engine/package/schemas/grid_loss_sys_corr_schema.py:1]: different file name?

**Review by JesperGomes — APPROVED** [review]

File: source/databricks/calculation-engine/package/schemas/grid_loss_sys_corr_schema.py
Comment (johevemi) [reply source/databricks/calculation-engine/package/schemas/grid_loss_sys_corr_schema.py:1]: it is not used so i have removed the file

**Review by JesperGomes — APPROVED** [review]

**Pull request merged**